.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+  - "12"
+  - "10"
+  - "8"
+  - "6"
+  - "0.12"
+  - "0.10"
```

Comment:
I omitted node 4 b/c it is in conflict with request and doesn't seem extremely important to you. I'm not sure if rolling request back a few versions to support node 4 is worth the trade off of using an old version of a lib.  LMK if you have an opinion or suggestion.

Node 0.8 is really old and we agreed to drop support for that.

Reply:
For me, node4 installs and tests just fine locally, so I'm not sure what's going wrong here. I do have node 4 in prod. Maybe this is a yarn issue?

Reply:
weird. Travis complaining about a sub-dep of request:

```
$ yarn
yarn install v1.3.2
[1/4] Resolving packages...
inimist@0.0.8inimist@0.0.8[2/4] Fetching packages...
error har-validator@5.1.3: The engine "node" is incompatible with this module. Expected version ">=6".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

Reply:
Yeah it must be Yarn doing version checks and npm not.

Reply:
This isn't yarn's fault. The module explicitly says it is only compatible with node 6+

https://github.com/ahmadnassri/node-har-validator/blob/master/package.json#L23

Reply:
`yarn install --ignore-engines` would probably work if we could configure that

Reply:
This is a floating semver thing and yarn is actually doing the right thing here.

Reply:
Yeah I agree that yarn is doing the right thing based on how the engines is configured. har-validator is configured with a version 6 engine because it relies on a dev dependency (tap ^12). Tests do pass for har-validator on node 4 with tap ^11, so I believe it does work with node4.

Reply:
I got node 4 working by removed lockfiles. I'm gonna merge/squash this, then publish.

Reply:
👍